lib/generators/shopify_app/root_controller/templates/root_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class RootController < ApplicationController
+  include ShopifyApp::EmbeddedApp
+  include ShopifyApp::RequireKnownShop
+
+  def index
+    @shop_orgin = current_shopify_domain
```

Comment:
Should we ship an example index.html.erb for this controller to render? It looks like as it stands this code won't work because it'll be expecting one.

Reply:
I think the next step of this PR could be configuring an entire default view (including a sample SPA and index.html file to render it)